frontend/lib/src/components/widgets/DataFrame/hooks/useColumnLoader.ts

```diff
@@ -297,6 +297,11 @@ function useColumnLoader(
     element.useContainerWidth ||
     (notNullOrUndefined(element.width) && element.width > 0)
 
+  // Allow content wrapping if the configured row height is greater than 4rem
```

Comment:
Could you please make the comment more specific please and say _why_ `4rem` is the limit here? Where does this magic number come from?

Reply:
> arbitrarily chosen because it looks good

Yep, its arbitrarily chosen because it looked and felt good. I will add that as comment.

> Also, should it be theme-able in case someone changes the sizes of the overall app?

Thats why it is using rem to automatically adapt to the configured root font size (the only way to currently re-scale apps).